addons/l10n_ch/models/res_bank.py

```diff
@@ -3,6 +3,12 @@
 
 import re
 
+try:
+    from stdnum import iso11649
+except ImportError:
+    import stdnum
+    _logger.warning("stdnum.iso11649 is avalaible from stdnum >= 1.10. The one installed is %s. You cannot use l10n_ch creditor reference." % stdnum.__version__)
```

Comment:
`avalaible` -> `available`

Reply:
Also, _logger is undefined there. 

See:
https://github.com/odoo/odoo/blob/a7f7233e0eae8ee101d745a9813cba930fd03dcb/addons/l10n_ar/models/res_partner_bank.py#L4-L12

Reply:
That code in `l10n_ar` also creates an alternative `validate_cbu` function to be used and handles the case
https://github.com/odoo/odoo/tree/master/addons/l10n_ar/models/res_partner_bank.py#L43-L49
while here we just log a warning. 🤔

Reply:
@lordkrandel I don't know what is the best solution, but i can ship the needed functions (is_valid and compact) if you prefer

Reply:
The version of stdnum that is in the requirements in inferior to 1.10 (1.8), so indeed, since it's only 2 functions we need, it'd probably be better to add them directly in the module, and not rely on stdnum for now.

Reply:
removed dependency on sdtnum